package.json

```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "dotenv": "^8.2.0",
```

Comment:
Why did you set dotenv as a dev dependency... Some deployment environment don't have a way of setting environment variables...

Reply:
Hi @micaiah-effiong, because when I tried to do npm start it didn't work because it couldn't find the dependency.
This dependency it's necessary in dev.

Reply:
If you want I can revert this change. 😄